### PR TITLE
Add loadimpact/k6 plugin

### PR DESCRIPTION
**Pull request by grimoh**

### CODE REVIEW

**Comment by imbsky** [comment]
Which badge is correct, Travis CI or GitHub Actions? Link the correct badge to README.md.

**Comment by grimoh** [comment]
The correct badge is GitHub Actions.
Thank you.

File: README.md [hunk 1]
```diff
@@ -88,6 +88,7 @@ itself) to look for plugins.
 | Jsonnet   | [Banno/asdf-jsonnet](https://github.com/Banno/asdf-jsonnet) | [![Build Status](https://travis-ci.org/Banno/asdf-jsonnet.svg?branch=master)](https://travis-ci.org/Banno/asdf-jsonnet)
 | Java      | [halcyon/asdf-java](https://github.com/halcyon/asdf-java) | [![Build Status](https://travis-ci.org/halcyon/asdf-java.svg?branch=master)](https://travis-ci.org/halcyon/asdf-java)
 | Julia     | [rkyleg/asdf-julia](https://github.com/rkyleg/asdf-julia) | [![Build Status](https://travis-ci.org/rkyleg/asdf-julia.svg?branch=master)](https://travis-ci.org/rkyleg/asdf-julia)
+| k6        | [grimoh/asdf-k6](https://github.com/grimoh/asdf-k6) | [![Build Status](https://travis-ci.org/grimoh/asdf-k6.svg?branch=master)](https://travis-ci.org/grimoh/asdf-k6)
```
Comment (imbsky) [inline README.md]: Could you replace CI badge to GitHub Actions one?

Comment (grimoh) [reply README.md]: Fixed.

Comment (imbsky) [reply README.md]: Thanks.

**Comment by imbsky** [comment]
Thanks @grimoh for your contribution!

**Comment by imbsky** [comment]
(and sorry for the late response. I've been little busy recently...)

**Pull request merged**